app/controllers/buyers/applications_controller.rb

```diff
@@ -24,19 +25,16 @@ def index
     activate_menu :audience, :applications, :listing
 
     @states = Cinstance.allowed_states.collect(&:to_s).sort
-    accessible_services = current_account.accessible_services
-    @services = accessible_services.includes(:application_plans)
     @search = ThreeScale::Search.new(params[:search] || params)
-    @application_plans = current_account.application_plans.stock
-    @stock_and_custom_application_plans = current_account.application_plans.size
+    @services = accessible_services.includes(:application_plans)
```

Comment:
It looks like `#accessible_services` already sets `@service`. You may want to move the `includes(:application_plans)` to that method and remove this line.